Packs/Active_Directory_Query/Integrations/Active_Directory_Query/Active_Directory_Query.yml

```diff
@@ -26,6 +26,14 @@ configuration:
   required: true
   section: Connect
   displaypassword: Password
+- display: Error on empty arguments
+  additionalinfo: If checked, will raise an error on empty arguments in the ad-get-user command.
+  name: fail_on_empty_args
```

Comment:
Let's also add it in the bottom of the parameters under "Create user if does not exist"